tests/frontend/test_kibana_app/cypress/integration/step-definitions/filters/navigate-to-agent-module.and.js

```diff
@@ -0,0 +1,14 @@
+import { And } from 'cypress-cucumber-preprocessor/steps';
+import { clickElement, elementIsVisible, elementXpathIsVisible, clickXpathElement} from '../../utils/driver';
+import {
+    moreLink
+  } from '../../pageobjects/agents/agents.page';
+import { AGENT_MODULES } from '../../utils/mappers/agent-modules-mapper';
+   
+And('The user navigates to {}', (moduleName) => {
+  cy.wait(1000);
+  elementXpathIsVisible(moreLink);
+  clickXpathElement(moreLink);
+  elementIsVisible(AGENT_MODULES[moduleName]);
```

Comment:
clickElement will validate if the element is visible and clickable. is there a point in validating it before clicking?